src/ios/getHeadersInFolder.js

```diff
@@ -1,7 +1,7 @@
 const glob = require('glob');
 const path = require('path');
 
-const GLOB_EXCLUDE_PATTERN = ['node_modules/**', 'Examples/**', 'examples/**'];
+const GLOB_EXCLUDE_PATTERN = ['node_modules/**', 'Pods/**', 'Examples/**', 'examples/**'];
```

Comment:
Further discussion in #6 (this kind of stuff could go there as well, as per @ptmt suggestion `findFiles(type, folder)` function